Sources/Amplitude/Amplitude.m

```diff
@@ -606,15 +621,20 @@ - (void)logEvent:(NSString *)eventType withEventProperties:(NSDictionary *)event
     }];
 }
 
+- (void)theTest {
```

Comment:
Is this finalized? We should move the test to Test forlder.

Reply:
Oh dang it, I left it in there